kubernetes/pods.json

```diff
@@ -104,119 +130,292 @@
+            "type": "property"
+          },
+          {
+            "property": "_missing_",
+            "propertyValue": "kubernetes_container_name",
+            "type": "property"
+          },
+          {
+            "property": "_exists_",
+            "propertyValue": "kubernetes_cluster",
+            "type": "property"
+          },
+          {
+            "property": "_exists_",
+            "propertyValue": "kubernetes_pod_name",
+            "type": "property"
+          },
+          {
+            "property": "_exists_",
+            "propertyValue": "host",
+            "type": "property"
+          },
+          {
+            "property": "_exists_",
+            "propertyValue": "kubernetes_namespace",
+            "type": "property"
+          }
+        ],
+        "template": "NETWORK_TX = data(\"pod_network_receive_bytes_total\"{{#filter}}, filter={{{filter}}}{{/filter}}, extrapolation=\"last_value\", maxExtrapolations=2).sum(by=[\"kubernetes_cluster\", \"host\", \"kubernetes_pod_name\", \"kubernetes_namespace\"])\nNETWORK_RX = data(\"pod_network_transmit_bytes_total\"{{#filter}}, filter={{{filter}}}{{/filter}}, extrapolation=\"last_value\", maxExtrapolations=2).sum(by=[\"kubernetes_cluster\", \"host\", \"kubernetes_pod_name\", \"kubernetes_namespace\"])\nNETWORK_TOTAL = NETWORK_TX + NETWORK_RX",
+        "varName": "NETWORK_TOTAL"
       },
       "coloring": {
         "method": "quantile",
-        "minValue": 0,
-        "maxValue": 100
+        "minValue": 0
       }
     },
     {
-      "id": "disk.used",
+      "id": "kubernetes.network.errors",
       "type": "metric",
-      "displayName": "Disk used by containers",
-      "valueLabel": "Disk Use",
-      "valueFormat": "Bytes",
+      "displayName": "Network errors",
       "metricSelectors": [
-        "container_fs_usage_bytes"
+        "pod_network_receive_bytes_total",
+        "pod_network_transmit_bytes_total"
       ],
-      "description": "Amount of disk, in bytes, used by containers in this pod.",
+      "description": "Total number of network errors from this pod. ",
       "job": {
         "resolution": 60000,
-        "filters": [],
-        "template": "DISK_CONTAINER_USED = data(\"container_fs_usage_bytes\"{{#filter}}, filter={{{filter}}}{{/filter}}, extrapolation=\"last_value\", maxExtrapolations=2).sum(by=[\"kubernetes_cluster\", \"host\", \"kubernetes_pod_name\"])",
-        "varName": "DISK_CONTAINER_USED"
+        "filters": [
+          {
+            "property": "_missing_",
+            "propertyValue": "name",
+            "type": "property"
+          },
+          {
+            "property": "_missing_",
+            "propertyValue": "image",
+            "type": "property"
+          },
+          {
+            "property": "_missing_",
+            "propertyValue": "cluster",
+            "type": "property"
+          },
+          {
+            "property": "_missing_",
+            "propertyValue": "kubernetes_pod_namespace",
```

Comment:
What is the point of filtering out MTSs with this old dimension?  To filter out old data?